Gruntfile.js

```diff
@@ -96,6 +96,10 @@ module.exports = function(grunt) {
     'env:dev',
   ]);
 
+  grunt.registerTask('unit', [
+    'test_unit'
+  ]);
```

Comment:
It was to make it the same as webapp. We may want to, at some point, do a pass over our various grunt commands and make them the same everywhere.